readthedocs/proxito/views/serve.py

```diff
@@ -673,3 +675,45 @@ def changefreqs_generator():
 
 class ServeSitemapXML(SettingsOverrideObject):
     _default_class = ServeSitemapXMLBase
+
+
+class ServeStaticFiles(CachedView, CacheTagsMixin, View):
```

Comment:
Should we rename `CachedView` as well? It seems very generic. Perhaps `CDNCacheControlMixin` or similar?